install.sh

```diff
@@ -151,6 +151,21 @@ install_zypper_packages( )
     done
 }
 
+install_msgpack_from_source( )
+{
+    if [[ ! -d "${build_dir}/deps/msgpack-c" ]]; then
+      pushd .
+      mkdir -p ${build_dir}/deps
+      cd ${build_dir}/deps
```

Comment:
I would recommend always quoting arguments containing shell variables (if they're supposed to be a single argument). `$build_dir` hopefully doesn't contain spaces anyways, but it's a good habit to get into.
```suggestion
      mkdir -p "${build_dir}/deps"
      cd "${build_dir}/deps"
```

Reply:
Good catch. It was a straight copy, so I wasn't looking too closely at that kind of thing. I'll add them in.